### PR TITLE
Photopicker

### PR DESCRIPTION
**Pull request by coreyadkins**

### CODE REVIEW

File: photopicker/photopicker.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+if (!flickrAPI) {
+  var flickrAPI = {};
```
Comment (schmamps) [inline photopicker/photopicker.js:4]: Another way to get around eslint is with`/* global flickrAPI, Promise */`

File: photopicker/photopicker.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    url: URL_STEM,
+    async: false
+  });
+  return JSON.parse(request.responseText);
+}
+
+/**
+ * Gets the coordinates for the given photo from flickr API
+ */
+function getPhotoLocation(photoid) {
```
Comment (schmamps) [inline photopicker/photopicker.js]: camelCase _cough_ 😄

File: photopicker/photopicker.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  });
+  return JSON.parse(request.responseText);
+}
+
+/**
+ * Picks a random photo from the list, then creates a URL to access the .jpg of
+ * the photo.
+ */
+function getPhotoLocAndURL(photosByPlace) {
+  var photo = _.sample(photosByPlace.photos.photo);
```
Comment (schmamps) [inline photopicker/photopicker.js:102]: Does using `_.sample()` prevent duplicates?

File: photopicker/photopicker.js [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+/**
+ * Main function- grabs a random photo off Flickr API with its location
+ * coordinates, and the country it was taken in.
+ */
+
+function getPhoto() {
+  var topCountriesList = getTopCountriesList();
+  var placesByCountry = getPlacesByCountry(topCountriesList);
+  if (placesByCountry[0].places.place.length === 0) {
```
Comment (schmamps) [inline photopicker/photopicker.js:127]: I think empty arrays and zero (`[]` and `[].length`) are also equal to`false`.

Don't fix, just saying.

File: photopicker/photopicker.js (hunk 2 above)
Comment (coreyadkins) [inline photopicker/photopicker.js]: That's how those particular argument are called as in the API parameters. Wouldn't recognize them if they were in camelCase

Comment (schmamps) [inline photopicker/photopicker.js]: `function getPhotoLocation(photoid)`

Comment (coreyadkins) [inline photopicker/photopicker.js]: Good point.

**Pull request merged**